src/server.c

```diff
@@ -3486,6 +3490,9 @@ void resetCommandTableStats(void) {
         c->calls = 0;
         c->rejected_calls = 0;
         c->failed_calls = 0;
+        if(c->latency_histogram) {
+            hdr_reset(c->latency_histogram);
```

Comment:
I have a minor preference for resetCommand resetting the state back to the launch state, so should we instead free this and set it to NULL? Then there is an online way to reduce the memory cost.

Reply:
👍 agree. doing the change on the next commit